webapp/src/templates/partials/contacts_content.html

```diff
@@ -70,73 +69,43 @@ <h3 translate>{{card.label}}</h3>
-                heading="child.doc.name"
-                summary="child.doc.title"
-                primary-contact="child.isPrimaryContact"
-                is-muted="child.doc.muted"
-                display-muted="true"
-              />
-            </ul>
-            <ul ng-if="$ctrl.selected.doc.type === 'clinic'">
-              <mm-content-row ng-repeat="child in $ctrl.selected.children.persons"
-                id="child.doc._id"
-                route="'contacts'"
-                heading="child.doc.name"
-                dob="child.doc.date_of_birth"
-                primary-contact="child.isPrimaryContact"
-                task-count="child.taskCount"
-                is-muted="child.doc.muted"
-                display-muted="true"
-              />
-            </ul>
-            <ul class="action deceased">
-              <li ng-if="$ctrl.selected.children.deceased.length" class="content-row deceased">
-                <a ui-sref="contacts.deceased({ id: $ctrl.selected._id })">
-                  <span translate translate-values="{ count: $ctrl.selected.children.deceased.length }">contact.deceased.view</span>
-                  <i class="fa fa-chevron-right"></i>
-                </a>
-              </li>
-            </ul>
-          </div>
-        </div>
+      <div class="card children" ng-if="!$ctrl.selected.loadingChildren && !$ctrl.selected.children.length">
+        <div class="cell col" translate>contact.no.children</div>
       </div>
 
-      <div ng-if="$ctrl.selected.children.places">
-        <div class="card children">
-          <div class="action-header cell">
-            <h3 translate>{{$ctrl.selected.children.childPlacesLabel}}</h3>
+      <div class="card children" ng-repeat="group in $ctrl.selected.children">
+        <div class="action-header cell">
+          <div class="col-xs-10">
+            <h3 translate>{{group.type.group_key}}</h3>
           </div>
-          <ul>
-            <mm-content-row ng-repeat="child in $ctrl.selected.children.places"
-              id="child.doc._id"
-              route="'contacts'"
-              heading="child.doc.name"
-              is-muted="child.doc.muted"
-              display-muted="true"
-            />
-          </ul>
+          <div ng-if="group.type.person" mm-auth="can_view_tasks" class="col-xs-2 tasks-title" translate>Tasks</div>
         </div>
+
+        <ul>
+          <mm-content-row ng-repeat="child in group.contacts" ng-if="!child.deceased"
+            id="child.doc._id"
+            route="'contacts'"
+            heading="child.doc.name"
+            summary="child.doc.title"
+            primary-contact="child.isPrimaryContact"
+            dob="child.doc.date_of_birth"
+            task-count="child.taskCount"
+            is-muted="child.doc.muted"
+            display-muted="true"
+          />
+        </ul>
+        <ul class="action deceased">
+          <li ng-if="selected.deceasedCount > 0" class="content-row deceased">
```

Comment:
There is no `selected.deceasedCount`. 
There is a `$ctrl.selected.deceasedCount` that is always 0 (comment left on `ContactViewModelGenerator` above).

Reply:
Fixed!